test.sh

```diff
@@ -8,7 +8,7 @@ PYTHON=python3
 
 if [[ -f /etc/redhat-release ]]; then
   . /etc/os-release
-  if [[ $VERSION_ID == 8 ]] ; then
+  if [[ $VERSION_ID == "8.10" ]] ; then
```

Comment:
I don't think there will be a 8.11 but this feels more correct. It'll fail with EL 80, but we'll see when we get there ;)
```suggestion
  if [[ $VERSION_ID == 8* ]] ; then
```